delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/RetryStrategy.scala

```diff
@@ -29,33 +27,40 @@ import scala.util.control.NonFatal
 final case class RetryStrategy[E](
     config: RetryStrategyConfig,
     retryWhen: E => Boolean,
-    onError: (E, RetryDetails) => BIO[E, Unit]
+    onError: (E, RetryDetails) => IO[Unit]
 ) {
-  val policy: RetryPolicy[BIO[E, *]] = config.toPolicy[E]
+  val policy: RetryPolicy[IO[*]] = config.toPolicy[E]
```

Comment:
Maybe another one for later - but do we do anything bespoke enough to need this wrapper? If not we could use cats-retry directly

Reply:
Probably but trying to no-brain this over the line